test/passes/simplify-locals_all-features.wast

```diff
@@ -1698,29 +1698,15 @@
     )
   )
 
-  (func $rethrow-trap (local $0 i32)
-    ;; This dead local.set cannot be replaced with a nop because rethrow can
-    ;; trap.
```

Comment:
Is this test no longer relevant because `rethrow` only trapped when given a null exnref?

Reply:
Yes.